### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by glozxi**
Add remind feature to UG.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -262,6 +263,25 @@ Calculates and displays the total revenue based on all the completed events.
...
+
+**Example**:
+
+Assume the current time is 22-03-2023 11:00.
+
+- `remind 2`
+  - Events that start within 2 days will be displayed. These are events that start on:
+    * 22-03-2023 after 11:00
+    * 23-03-2023
+    * 24-03-2023
```
Comment (ExtraShotLatte) [inline docs/UserGuide.md:283]: Is the timing a strict 24 hours based per day or will events on 24 after 11:00 be displayed if the specified `DAYS` is 2?

Good to clarify!

**Review by ExtraShotLatte — CHANGES_REQUESTED** [review]
Some clarity needed for the UG!

File: docs/UserGuide.md (hunk 1 above)
Comment (glozxi) [reply docs/UserGuide.md:283]: Will do 👍

File: docs/UserGuide.md [hunk 2]
```diff
@@ -256,12 +239,36 @@ Edits the specified event from the event book.
...
+**Example**:
+
+Assume the current date and time is 22-03-2023 11:00.
+
+- `remind 2`
+  - Events that start within 2 days will be displayed. These are events that start on:
+    * 22-03-2023 after 11:00
+    * 23-03-2023 the whole day
+    * 24-03-2023 the whole day
+  - Note that 24-03-2023 is within three days of 22-03-2023, so events on 24-03-2023 that start more than 48 hours from the current date and time are displayed.
```
Comment (ExtraShotLatte) [inline docs/UserGuide.md:270]: LGTM! Good clarity!

**Review by ExtraShotLatte — APPROVED** [review]
LGTM!

**Pull request merged**